src/XamlStyler/StylerService.cs

```diff
@@ -19,18 +21,65 @@ namespace Xavalon.XamlStyler
 {
     public class StylerService
     {
+        private readonly string[] ignoredNamespacesInOrdering = new string[]
+        {
+            "http://schemas.microsoft.com/expression/blend/2008",
```

Comment:
We should probably add "http://xamarin.com/schemas/2014/forms/design" as well to cover [Xamarin.Forms](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/xaml/xaml-previewer/design-time-data)